.pre-commit-config.yaml

```diff
@@ -8,6 +8,15 @@ repos:
         files: environment
         types: [yaml]
         additional_dependencies: ["ruamel.yaml"]
+      - id: nextflow-lint
+        name: Lint Nextflow scripts and fix formatting
+        entry: nextflow lint
+        language: system
+        files: '\.nf$|nextflow\.config$'
+        args: ["-format", "-sort-declarations", "-spaces", "4", "-harshil-alignment", "-output", "concise"]
```

Comment:
I'd much prefer a config file 😬 https://github.com/nextflow-io/nextflow/issues/5934

Though I guess we can always add that later.

Reply:
I would as well, but perfect is the enemy of good 😉